fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -50,38 +50,49 @@ type NavigationProps = CompositeScreenProps<
 // ALSO: for now the write_in feature is disabled (2022-03-16, Tyratox)
 type NewQuestion = Omit<QuestionState, 'id' | 'write_in'>;
 
-const EMPTY_QUESTION: NewQuestion = {
-  question: '',
-  voting_method: VOTING_METHOD,
-  ballot_options: [''],
-};
-
 const MIN_BALLOT_OPTIONS = 2;
 
 /**
  * Some helper functions
  */
 
 /**
- * Checks whether a given newly created question is invalid
+ * Checks whether a question with some ballots options is invalid
  * @param question The question to check
  */
-const isQuestionInvalid = (question: NewQuestion): boolean =>
-  question.question === '' || question.ballot_options.length < MIN_BALLOT_OPTIONS;
+const isNonEmptyQuestionInvalid = (question: NewQuestion): boolean =>
+  question.question === '' && question.ballot_options.length !== 0;
 
 /**
- * Checks whether a question title is not unique within a list of questions
+ * Checks whether the question list is invalid
+ * @param questions The question list to check
+ */
+const areQuestionsValid = (questions: NewQuestion[]): boolean => questions.length > 0;
+
+/**
+ * Checks whether the ballot options of the question are invalid
+ * @param question the question that contains the ballot options to check
+ */
+const areBallotOptionsInvalid = (question: NewQuestion): boolean =>
+  question.question !== '' &&
+  (new Set(question.ballot_options).size !== question.ballot_options.length ||
+    question.ballot_options.length < MIN_BALLOT_OPTIONS);
+
+/**
+ * Checks whether a question title is not unique withcreateEin a list of questions
  * @param questions The list of questions
  */
 const haveQuestionsSameTitle = (questions: NewQuestion[]): boolean => {
```

Comment:
You are checking whether they are unique, right?
```suggestion
const haveUniqueQuestionTitles = (questions: NewQuestion[]): boolean => {
```